08 Health HUD/gamedata/scripts/modxml_dart_health_hud.script

```diff
@@ -78,9 +78,22 @@ function parseDXMLCustomMsgs(DXML, xmlAspectRatio)
       return
     end
 
+    -- Save BHS parent container y position for fixing news positioning in modxml_dart_news
+    if name ~= "body_health_system" then
+      local hud_health_system_offset = baseHeight + offset.y - 1 * (HUD_BOUNDS.b - attrs.y)
+      DART.hud_health_system_y = hud_health_system_offset - (baseHeight + offset.y - scale * (HUD_BOUNDS.b - attrs.y))
```

Comment:
Taking the diff of if scale was `1` and the new scale to find the dynamic value of `y`. This `DART.hud_health_system_y` is then used later to move the News out of the way of BHS